<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Principal</class>
 <widget class="QMainWindow" name="Principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>CrossCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Mi Paint</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Recursos.qrc">
    <normaloff>:/Principal</normaloff>:/Principal</iconset>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionNuevo"/>
    <addaction name="actionGuardar"/>
    <addaction name="separator"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuEstilo">
    <property name="title">
     <string>Estilo</string>
    </property>
    <addaction name="actionColor"/>
    <addaction name="actionAncho"/>
   </widget>
   <widget class="QMenu" name="menuForma">
    <property name="title">
     <string>Forma</string>
    </property>
    <addaction name="actionLineas"/>
    <addaction name="actionLibre"/>
    <addaction name="actionRect_ngulo"/>
    <addaction name="actionCircunferencias"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuEstilo"/>
   <addaction name="menuForma"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionNuevo">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/Nuevo</normaloff>:/Nuevo</iconset>
   </property>
   <property name="text">
    <string>Nuevo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/Guardar</normaloff>:/Guardar</iconset>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/Salir</normaloff>:/Salir</iconset>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionColor">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/Colores</normaloff>:/Colores</iconset>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
  </action>
  <action name="actionAncho">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/Tamaño</normaloff>:/Tamaño</iconset>
   </property>
   <property name="text">
    <string>Ancho</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+A</string>
   </property>
  </action>
  <action name="actionLineas">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/Lineas</normaloff>:/Lineas</iconset>
   </property>
   <property name="text">
    <string>Líneas</string>
   </property>
  </action>
  <action name="actionLibre">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/Libre</normaloff>:/Libre</iconset>
   </property>
   <property name="text">
    <string>Libre</string>
   </property>
  </action>
  <action name="actionRect_ngulo">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/Rectangulo</normaloff>:/Rectangulo</iconset>
   </property>
   <property name="text">
    <string>Rectángulo</string>
   </property>
  </action>
  <action name="actionCircunferencias">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/Circulos</normaloff>:/Circulos</iconset>
   </property>
   <property name="text">
    <string>Circunferencias</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Recursos.qrc"/>
 </resources>
 <connections/>
</ui>
